From d3f0766b96754e2d36d8e36076ada09093731ca9 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Mon, 2 Jul 2012 22:21:06 +0300
Subject: [PATCH 1/3] SPL/Parser2.hs: fix 'foldl' symbol clash (base-4.4)

[ 6 of 11] Compiling SPL.Parser2      ( SPL/Parser2.hs, dist/build/splm/splm-tmp/SPL/Parser2.o )

SPL/Parser2.hs:217:43:
    Ambiguous occurrence `foldl'
    It could refer to either `Prelude.foldl',
                             imported from `Prelude' at SPL/Parser2.hs:1:8-18
                             (and originally defined in `GHC.List')
                          or `M.foldl', imported from `Data.Map' at SPL/Parser2.hs:3:1-33

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 SPL/Parser2.hs |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/SPL/Parser2.hs b/SPL/Parser2.hs
index 2a05da2..c608b43 100755
--- a/SPL/Parser2.hs
+++ b/SPL/Parser2.hs
@@ -1,6 +1,6 @@
 module SPL.Parser2 (P (..), Syntax (..), SynParams (..), parse, res) where
 
-import Data.Map as M hiding (map)
+import qualified Data.Map as M
 -- UNUSED import System.IO.Unsafe
 
 data SynMark =
-- 
1.7.8.6

